runtime/jcl/common/jclcinit.c

```diff
@@ -116,6 +116,13 @@ jint computeFullVersionString(J9JavaVM* vm)
 			j2se_version_info = "12.?";
 		}
 		break;
+	case J2SE_V13:
+		if ((J2SE_VERSION(vm) & J2SE_RELEASE_MASK) == J2SE_V13) {
+			j2se_version_info = "13";
```

Comment:
The next step would be to build this string automatically from the version, but I suggest we leave that to another PR.